From 2be7a8dc690944849a60fe05b506cc06300f5ca5 Mon Sep 17 00:00:00 2001
From: Harald Sitter <sitter@kde.org>
Date: Tue, 23 Jun 2026 11:41:37 +0200
Subject: [PATCH 2/4] UPSTREAM: about: warn when using the single gpu fallback
 path

this means both vk and egl are broken. we want to eventually learn about
this

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 kcms/about-distro/src/GPUEntryFactory.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kcms/about-distro/src/GPUEntryFactory.cpp b/kcms/about-distro/src/GPUEntryFactory.cpp
index 0d891cc7..0be0d670 100644
--- a/kcms/about-distro/src/GPUEntryFactory.cpp
+++ b/kcms/about-distro/src/GPUEntryFactory.cpp
@@ -237,6 +237,8 @@ std::optional<std::vector<GPUEntry::Device>> gpus()
 
 QString fancyOpenGLRenderer()
 {
+    qWarning() << "Falling back to QOpenGLContext for renderer name. This is a fallback and may not be accurate.";
+
     QOpenGLContext context;
     QOffscreenSurface surface;
     surface.create();
-- 
2.56.0

